docs/snippets/dashboard.json

```diff
@@ -83,10 +40,7 @@
       "type": "row"
     },
     {
-      "datasource": {
-        "type": "prometheus",
-        "uid": "${DS_PROMETHEUS}"
-      },
```

Comment:
This had to be changed manually, this would throw an error "data source Prometheus not found" - this is due to the `uid` being specified here. 
I found out through experimentation that this can be changed to just `$datasource`.